src/LanguageServer/Impl/LanguageServer.cs

```diff
@@ -145,6 +145,7 @@ private void OnUnregisterCapability(object sender, UnregisterCapabilityEventArgs
                 settings.diagnosticPublishDelay = GetSetting(analysis, "diagnosticPublishDelay", 1000);
                 settings.symbolsHierarchyDepthLimit = GetSetting(analysis, "symbolsHierarchyDepthLimit", 10);
                 settings.symbolsHierarchyMaxSymbols = GetSetting(analysis, "symbolsHierarchyMaxSymbols", 1000);
+                settings.analysis.diagnostics = GetSetting(analysis, "diagnostics", false);
```

Comment:
Default should be `true` as the setting is not in the `package.json` yet

Reply:
Okay, though that will mean that VSC will double-up on a couple of diagnostics since they still run pylint for everything.